bundles/org.openhab.binding.knx/README.md

```diff
@@ -267,6 +267,18 @@ With `*-control` channels, the state is not owned by any device on the KNX bus,
 The element `dpt` is  highly recommended and may change to a mandatory element in future versions.
 If omitted, the corresponding default value will be used (see the channel descriptions above).
 
+## Special DPTs
+
+OpenHAB supports all DPTs supported by the corresponding release of Calimero library.
+
+Additional DPTs have been introduced to add functionality:
+
+| DPT           | Description                                                 | Remark     |
+|---------------|-------------------------------------------------------------|------------|
+| DPT 232.60000 | DPT 232.600 with HSB instead of RGB data (see below)        | read/write |
+| DPT 235.001   | Composed DPT 235.001, first element ActiveEnergy (Wh)       | read only  |
+| DPT 235.61001 | Composed DPT 235.001, second element Tariff (plain number)  | read only  |
```

Comment:
235.001 is the defined DPT, correct? We added 00 as custom type for 232.600, so maybe we should use 235.00100 here. If we need another one, we could then increment the last digit.

Reply:
I thought about a good naming scheme as well, as more of those multipart DPTs may be needed in the future.

My idea for the subtypes is as follows:

- 60000 is base for extensions, this is in the area of manufacturer specific DPTs, .
- the second digit refers to the element of the multipart message
- digits 3-5 are the old subtype (as there are subtypes like .600 we need 3 digits).

The original DPT is kept for the main content if the multipart message (in this case ActiveEnergy).

232.60000 does not follow this scheme, but has been introduced before.

Reply:
I'm fine with the explanation.